include/nanvix/region.h

```diff
@@ -42,7 +43,7 @@
 	#define REGION_SIZE   ((size_t)REGION_PGTABS*MREGIONS*PGTAB_SIZE)
 
  	/* Mini region dimensions. */
-	#define NR_MINIREGIONS (32) /* # Mini regions.            */
+	#define NR_MINIREGIONS (128) /* # Mini regions.            */
```

Comment:
Yes we saw it together. As I understood the kernel, as you can see in nanvix/include/nanvix/config.h, NR_REGIONS is 128. Or, if NR_MINIREGIONS is set to 32, it means that NR_REGIONS is limited to 32. So it's for homogenisation. In fact change is not useful for my code and could be dropped, it's juste to harmonize kernel code.

Reply:
@turbolay awesome... I had forgotten that...

Reply:
Since each process should have at least one miniregion, I think 64 is a good number, because follows the number of process and addresses 4GB.

128 miniregions give us 8GB of memory (4MB/pte * 16 REGION_PGTABS * 128 NR_MINIREGIONS) , which is not currently supported. 

One curious thing is that Nanvix can't address 4GB, due to the distance of heap/stack be 512MB, so the virtual and physical addresses need to be readjusted if 4GB is necessary.